client/core/trade.go

```diff
@@ -2879,6 +2879,24 @@ func (c *Core) sendRedeemAsync(t *trackedTrade, match *matchTracker, coinID, sec
 					},
 				})
 			}
+			if errors.As(err, &msgErr) && msgErr.Code == msgjson.RPCUnknownMatch {
```

Comment:
Looks like the same thing could happen when we send `msgjson.InitRoute` in `sendInitAsync`. Do we need to self-revoke there too?

Reply:
I was able to cause the same error with init, so yes.